addon/components/bs3/bs-form.js

```diff
@@ -1,9 +1,12 @@
-import { computed } from '@ember/object';
+import { computed, assert } from '@ember/object';
```

Comment:
should be `import { assert } from '@ember/debug';`
See https://github.com/ember-cli/ember-rfc176-data